### PR TITLE
Chunyin development

### PR DESCRIPTION
**Pull request by jevrii**

### CODE REVIEW

File: walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java [hunk 1]
```diff
@@ -37,7 +37,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Convert the server stats to JSON
     ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
-    String json = convertToJson(serverStats);
+    // String json = convertToJson(serverStats);
+    String json = convertToJsonUsingGson(serverStats);
```
Comment (ziyeqinghan) [inline walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java:41]: These changes should be in next commit when adding gson dependency into pom.xml. Please keep in mind next time. 

Meanwhile, remove the unused code other than comment the code.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,37 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadContent() {
+    // addName();
```
Comment (ziyeqinghan) [inline portfolio/src/main/webapp/script.js:31]: Remove these lines.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,37 @@ function addRandomGreeting() {
...
+    addComments();
+}
+
+function addName() {   
+//   const response = await fetch('/data');
+//   const name = await response.text();
```
Comment (ziyeqinghan) [inline portfolio/src/main/webapp/script.js:37]: ditto.

Comment (jevrii) [reply portfolio/src/main/webapp/script.js]: removed in a52768f

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (jevrii) [reply portfolio/src/main/webapp/script.js]: - Seems that each div cannot add its own `onload` attribute? https://stackoverflow.com/questions/4057236/how-to-add-onload-event-to-a-div-element
- As `onload` in `<body>` only supports 1 function, I decided to use the `loadContent()` function to call multiple functions. Probably there exists a better way...

Comment (ziyeqinghan) [reply portfolio/src/main/webapp/script.js]: Currently just add `onload` in `<Body>` SGTM.

**Pull request merged**